### PR TITLE
feat(add new features): add the number of attendees in the event

### LINKED ISSUE #61: the event attendees
**Task title**
the event attendees

**Task description**
show the amount of people that will attend the event in the event card 

**Subtasks**
Put the subtasks here if any (optional)

**Screenshots**
Capture d’écran 2023-11-04 075623

**Link to the component on Figma**
[Put the link here](https://www.figma.com/file/IEhYhDLnrHmUuYercEZUjA/Pebble-work?type=design&node-id=1878-5247&mode=design&t=fpGP9BLmonMH4qs0-4)

### PR DESCRIPTION
**Pull request by oualidelh**
#61

# Related Issue

- Resolve #61

### CODE REVIEW

**Review by Khalil-NOUI — COMMENTED** [review]
great Job, love the button design, unfortunately i think the actual functionality of the join feature is now broken, can you please check it.
in overall Amazing work and love to understand the logic behind it further more.
Keep Up the good Work, Oualid.

File: package-lock.json
Comment (HachemBouhadede) [inline package-lock.json:1]: not sure why this is marked as changed

File: src/components/eventsPageComponents/EventCard.jsx
Comment (HachemBouhadede) [inline src/components/eventsPageComponents/EventCard.jsx:1]: understandable logic

**Review by HachemBouhadede — APPROVED** [review]
understable logic, good job ! <3

File: src/components/eventsPageComponents/EventCard.jsx
Comment (oualidelh) [reply src/components/eventsPageComponents/EventCard.jsx:1]: understandable logic !!!
 what do you mean ?

**Pull request merged**